src/views/StandaloneDirectorsFiling.vue

```diff
@@ -510,16 +544,88 @@ export default {
   methods: {
     ...mapActions(['setFilingData']),
 
+    async fetchDraftFiling (): Promise<void> {
```

Comment:
I moved this up from below, both to be consistent with the other filings and because it's used first.